README.md

```diff
@@ -20,7 +20,7 @@ Simply Love is compatible with current releases from the official StepMania proj
 **Compatible Releases**<br>
 ✅ [StepMania 5.0.12](https://github.com/stepmania/stepmania/releases/tag/v5.0.12)<br>
 ✅ [StepMania 5.1-b2](https://github.com/stepmania/stepmania/releases/tag/v5.1.0-b2)<br>
-✅ [OutFox 5.3 alpha 4.12.0](https://projectoutfox.com/releases/5.3.0-alpha-4.12.0) (preliminary support)<br>
```

Comment:
Do we mention that previous builds of OutFox (i.e. when it was still known as StepMania 5.3) are no longer supported?

Reply:
updated